src/components/Item/Item.jsx

```diff
@@ -10,21 +10,27 @@ class Item extends Component {
   constructor() {
     super();
     this.state = {
-      isCreatable: false,
-      topic: []
+    topic: [],
+    isCreatable: false,
+    isDeletable: false
     };
     // These bindings are necessary to make `this` work in the callback
     this._assignTopic = this._assignTopic.bind(this);
     this._removeTopic = this._removeTopic.bind(this);
     this._fetchItem = this._fetchItem.bind(this);
-    this._checkIsCreatable = this._checkIsCreatable.bind(this);
+    this.checkIsCreatable = this.checkIsCreatable.bind(this);
+    this.buttonValidateCreate =this.buttonValidateCreate.bind(this);
+    this.buttonCreate = this.buttonCreate.bind(this);
+    this.handleDeleteClick = this.handleDeleteClick.bind(this);
+    this.handleValidateDeleteClick = this.handleValidateDeleteClick.bind(this);
+    this.deleteAttribut = this.deleteAttribut.bind(this);
```

Comment:
"Each line of code [should be] written in English": `deleteAttribut` -> `deleteAttribute`.